lnwallet/channel.go

```diff
@@ -1790,6 +1790,30 @@ func (lc *LightningChannel) restoreStateLogs(
 		outgoingLocalAddHeights[l.HtlcIndex] = localCommitment.height
 	}
 
+	// If we have any unsigned acked updates to sign for, then the add is no
```

Comment:
commit message must be updated with the latest change.

Reply:
commit message updated